### ISSUE TITLE
lab 21

### ISSUE THREAD

**Issue by elixir123102030** [elixir123102030/week3#6]
import pandas as pd
import numpy as np
from sklearn.linear_model import LinearRegression
from sklearn.metrics import r2_score

x = pd.read_csv("data/day21_housing.csv")

y = x["price"]
u = x.drop("price", axis=1)

x["pp"] = np.where(x["sqft"] == 0, 0, x["price"] / x["sqft"])

x["old"] = 2026 - x["year_built"]
x["dens"] = np.where(x["rooms"] == 0, 0, x["sqft"] / x["rooms"])

a = LinearRegression()
a.fit(u, y)
r1 = r2_score(y, a.predict(u))

v = x.drop("price", axis=1)

a.fit(v, y)
r2 = r2_score(y, a.predict(v))

print(r1)
print(r2)
